CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [0.0.16] - 2020-01-22
 ### Added
 - Core APIs from ZAP version 2.8.0.
```

Comment:
n-1 ?

Reply:
Those were added in 2.8.0, though I guess now it's irrelevant with the 2.9.0 release. Will tweak.

Reply:
Tweaked.